### ISSUE TITLE
Couldn't import tesserocr, because locale check error

### ISSUE THREAD

**Issue by atuyosi** [sirfz/tesserocr#137]
I got a import error.

```
import tesserocr
!strcmp(locale, "C"):Error:Assert failed:in file baseapi.cpp, line 203
Abort trap: 6
```

This error was caused by locale check.

Please see [commit](https://github.com/tesseract-ocr/tesseract/commit/3292484f67af8bdda23aa5e510918d0115785291#diff-903b81bdfe923135ee1d03800c400ed8) .

Simple workaround here .

```
import locale
locale.setlocale(locale.LC_ALL, 'C')
import tesserocr
```

I think that it is necessary to add the code somewhere in the appropriate place.

Environment:

- macOS 10.13.6
- Python 3.6.5
- tesserocr  2.3.0
- tesseract 4.0.0-beta.4-20-ge9b4e

In addition, I avoid install error by #129 workaroud.

**Comment by atuyosi** [comment]
As you said, the hardcoding is undesirable.

IMO, I think that it is necessary to ask the Cython community about handling environment variables.

cf. [24.2. locale — Internationalization services — Python 3.7.0 documentation](https://docs.python.org/3.7/library/locale.html#for-extension-writers-and-programs-that-embed-python)

FYI, other Language's solution.

[Various fixes for Tesseract 4 beta.3 · ropensci/tesseract@2784542](https://github.com/ropensci/tesseract/commit/2784542f7cd21ed99a9b5b43961179a4922e7383)

**Comment by sirfz** [comment]
Changing locale before and after calling `Init` seems reasonable so I'll go with that (thanks for the links). Would resetting the locale to something other than C affect the results of other API methods though?

**Comment by sirfz** [comment]
This has been fixed in tesseract 4.1
